zulip/integrations/slack/slackdata2zulipdata.py

```diff
@@ -250,8 +251,8 @@ def sanitize_token(token):
         msgs = json.load(open(slack_dir + '/%s/%s' % (channel, json_name)))
         for msg in msgs:
             text = msg['text']
-            if "has joined the channel" in text:
-                continue
+            # TODO directly ignore noisy messages from msg['subtype']
```

Comment:
i.e. `if msg['subtype'] in ["channel_join", "channel_leave", ...]: continue`?

Reply:
I will do this later.

Reply:
Nvm, I will leave this to you; I am focusing on stage 2.

Reply:
Okay, I'll update this.